.github/workflows/buid_and_release.yml

```diff
@@ -16,5 +16,20 @@ jobs:
     - uses: actions/checkout@v2
     - name: build
       run: make all
+    - name: Get current time
+      uses: josStorer/get-current-time@v2.0.2
+      id: current-time
+      with:
+        format: YYYY-MM-DD
     - name: Release
```

Comment:
Currently failing with:
```
  Error: request to https://uploads.github.com/repos/monarch-initiative/medgen/releases/111345495/assets?name=medgen-fd88bd77ee71d855a9d4cd3ff9ee27d7.obo& failed, reason: read ECONNRESET
```

I haven't seen this before. I'm gonna try running again. Otherwise I'll troubleshoot later.

Running locally using the GitHub CLI worked though.

Reply:
It ran successfully now. I think it was just because I ran a previous action as a test run (without uploading any actual files) that created a release with the same title. So I had to delete that first.